chapter05/red/exercise2.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+use List::Util qw( reduce );
+
+my $fn = shift @ARGV // 'coconet.dat';
+open my $fh, '<', $fn;
+
+my %stats;
+while( <$fh> ) {
+  next if /^\s*(#.*)?$/;
```

Comment:
```perl
if ( /^\s*(#.*)?$/ ) {
  next;
}
```
Makes me feel ill. It'd be alright if perl allowed braceless single statement conditional bodies.